doc/admin-guide/plugins/header_rewrite.en.rst

```diff
@@ -142,6 +142,23 @@ with the word ``else``. The following example illustrates this::
 The ``else`` clause is not a condition, and does not take any flags, it is
 of course optional, but when specified must be followed by at least one operator.
 
+You can also do an ``elif`` (else if) clause, which is specified by starting a new line
+with the word ``elif``. The following example illustrates this::
+
+    cond %{STATUS} >399 [AND]
+    cond %{STATUS} <500
+      set-status 404
+    elif
+      cond %{STATUS} =503
+        set-status 502
+    else
+      set-status 503
+
+Keep in mind that nesting the ``else`` and ``elif`` clauses is not allowed, but any
+number of ``elif`` clauses can be specified. We can consider these clauses are more
+powerful and flexible ``switch`` statement. In an ``if-elif-else`` rule, only one
+will be evaluate its operators.
```

Comment:
How about adding...

Similarly, each ``else`` and ``elif`` have the same implied :ref:`Hook Condition <Hook Conditions>` as the initial condition.

Reply:
Added.